src/openforms/ui/tests/test_header.py

```diff
@@ -24,9 +24,7 @@ def test_logo_label(self, get_solo_mock):
 
         header_node = response.html.find("header")
         link_node = header_node.find("a")
-        aria_label = link_node.get("aria-label")
```

Comment:
These were explicit a11y patches: #2428 and #1018

So please explain to me why this patch is correct and the other one wasn't, because now we're just flipping back and forwards between approaches.

Reply:
The aria-label was removed as a attempt to remove the duplicate calling out of the title (the screenreader would announce the anchor as "Back to website Back to website").

In the rapport it was suggested to remove the `title` and `aria-label` and change the content to "Back to website of [OrganisationName]", to solve this issue. Due to the tooltip functionality of `title`, I decided to only remove the `aria-label`, because the content and `title` attribute would give the same needed information to screen readers. But i just noticed that this doesn't solve the "double title calling" issues.

After some digging I found out that this apparently is a issues with specific screen readers in combination with specific browsers. Some browsers don't/won't play nice with some screen readers..? I our case, the Apple VoiceOver screenreader will read the anchor content twice regardless of the presence of `label` and `aria-title` (tested with Chrome Version 129.0.6668.101 and Safari Version 17.6). I haven't yet tested this with other screenreader.

For now i will return the `aria-label`, seeing how this doesn't remove information or disrups the screenreader.